### DISCUSSION TITLE
Filtering cascading table widgets choosing multiple lines

### DISCUSSION THREAD

**Discussion by Bitello** [Budibase/budibase#9309] [Help] [answered]
Discussed in https://github.com/Budibase/budibase/discussions/9033
Hi!

I've been digging around the forum and it's been really interesting as I've been able to figure out how to properly use some features.

In my previous discussion I learned how to filter cascading tables (Thanks to @andz-bb).

But now I need to choose multiple rows from a table and the tables below must be filtered according to the selected rows.

I researched a lot, tried several alternatives, but I couldn't. My knowledge is in-depth in TSql, but not in javascript...

Anyway... could anyone guide me on how I should proceed to perform this type of filtering?

Grateful!

**Comment by melohagan** [comment, marked as answer]
Hey @Bitello 

Does this answer help? https://github.com/Budibase/budibase/discussions/8759#discussioncomment-4207094

**Reply by Bitello** [reply]
Hello, @melohagan melohagan.

In your example, both pages have the same datasource.

Imagine a Sales table and a Products table.

If I select one or more Sales, the Products from the respective Sales will be displayed in the Products table

**Reply by melohagan** [reply]
Ah, filter by relationship is not available for the internal DB.
